backend/app/tests/assessment/test_export.py

```
@@ -0,0 +1,224 @@
+
+    def test_csv_format_returns_csv_bytes(self) -> None:
+        rows = self._make_rows()
+        payload, media_type = serialize_export_rows(rows, "csv")
+        assert media_type == "text/csv"
+        content = payload.decode("utf-8")
+        assert "score" in content
+
+    def test_csv_contains_all_rows(self) -> None:
+        rows = self._make_rows()
+        payload, _ = serialize_export_rows(rows, "csv")
+        lines = [l for l in payload.decode("utf-8").splitlines() if l.strip()]
```

Ruff (0.15.12)

[error] Ambiguous variable name: `l`

(E741)